docs/section5/Rule5-40.md

```diff
@@ -24,18 +24,16 @@
 
 - For each building segment in the Baseline model: `for building_segment_b in B_RMR.building.building_segments:`
 
-  - For each thermal_block in building segment: `for thermal_block_b in building_segment_b.thermal_blocks:`
+  - For each zone in thermal block: `for zone_b in building_segments.zones:`
```

Comment:
Should be "For each zone in building_segment: `for zone_b in building_segment_b:`